src/providers/Wallet.js

```diff
@@ -13,23 +20,122 @@ function useWalletAugmented() {
 
 // Adds Ethers.js to the useWallet() object
 function WalletAugmented({ children }) {
+  // This new state is neede to avoid showing the user the profile signature request on every network change
+  const [chainReseted, setChainReseted] = useState(false)
+  const [chainId, setChainId] = useState(-1)
```

Comment:
I believe this `chainId` state variable is not relevant anymore with `preferredNetwork`. In places, we would like a reference to the chainId we just use `wallet.chainId`

Reply:
good catch gabi, that state stayed old since when i started to do the first iteration and forgot to delete it